testsuite/features/step_definitions/command_steps.rb

```diff
@@ -872,13 +870,12 @@
 end
 
 When(/^I install package tftpboot-installation on the server$/) do
-  node = get_target("server")
-  output, _code = node.run("find /var/spacewalk/packages -name tftpboot-installation-SLE-15-SP2-x86_64-*.noarch.rpm")
+  output, _code = $server.run('find /var/spacewalk/packages -name tftpboot-installation-SLE-15-SP2-x86_64-*.noarch.rpm')
```

Comment:
Given that this local copy of a rpm exists only for this specific package so far, there is not much point in generalizing - yagni.

In any case, it would be for another time.

Reply:
Note that in the change I proposed, we will not need to update this step, when we use sles15sp3 version ;)

Reply:
I think @cbosdo is the author of that code... Ping Cédric ?

Reply:
the problem that is solved here is that we have multiple versions of the `tftpboot-installation` package, we need the one matching the distro of the VM to create using tftpboot and with the latest version.

Note that we are installing a package that may not fit the OS of the server... which is why we are installing from the reposync'ed packages.

Reply:
you will need to update this package to 15SP3 only when updating the distro in those places:

  * https://github.com/uyuni-project/uyuni/blob/master/testsuite/features/secondary/minkvm_guests.feature#L374
  * in the reposync white list

Reply:
Would you mean that doing :
```
output, _code = $server.run('find /var/spacewalk/packages -name tftpboot-installation-*.noarch.rpm')
```
Will show multiple rpms? If that's the case, then ok, otherwise, I think that removing the OS version will help here, so we don't need to keep maintaining this step definition, everytime we change the version.

Reply:
I see, ok.

```mgr-srv-41:~ # find /var/spacewalk/packages -name tftpboot-installation-*x86_64*.noarch.rpm
/var/spacewalk/packages/NULL/868/tftpboot-installation-SLE-15-SP1-x86_64/14.424-1.9/noarch/868d1f332cdb4bad06bb2666b39fa934b711e98029214a40d32ead4ac04bf4bf/tftpboot-installation-SLE-15-SP1-x86_64-14.424-1.9.noarch.rpm
/var/spacewalk/packages/NULL/933/tftpboot-installation-SLES-12-SP5-x86_64/14.337.16-1.26/noarch/933ae663d935f37857799ceb79d3b2a660af40efb783d3ac40e3f331586f4200/tftpboot-installation-SLES-12-SP5-x86_64-14.337.16-1.26.noarch.rpm
/var/spacewalk/packages/NULL/e37/tftpboot-installation-SLE-15-x86_64/14.385-3.3.2/noarch/e37688fd6b8d9b6342d698a82dc36f79f31f02763fde70875648da5e7af45c0e/tftpboot-installation-SLE-15-x86_64-14.385-3.3.2.noarch.rpm
/var/spacewalk/packages/NULL/ab8/tftpboot-installation-SLE-15-SP2-x86_64/14.477-3.11.1/noarch/ab824181a960e97275eb2e499835739bc57a273b831aebe928ca0fac7226c9dd/tftpboot-installation-SLE-15-SP2-x86_64-14.477-3.11.1.noarch.rpm
/var/spacewalk/packages/NULL/112/tftpboot-installation-SLE-15-x86_64/14.373-1.14/noarch/1126bad34e177daa3558ad05a10075ef4449a3a86f9ef9369314313cd0acb092/tftpboot-installation-SLE-15-x86_64-14.373-1.14.noarch.rpm
/var/spacewalk/packages/NULL/854/tftpboot-installation-SLE-15-SP2-x86_64/14.472-3.3.2/noarch/854a9de1228963a3c00faa5403995acd5cb0ccbdf2e8b890da3152bf2cbf2797/tftpboot-installation-SLE-15-SP2-x86_64-14.472-3.3.2.noarch.rpm
/var/spacewalk/packages/NULL/60d/tftpboot-installation-SLE-15-SP1-x86_64/14.427.1-3.6.2/noarch/60dc58e90bdb47a901e19598ba95450b75929be7afc0ca741afd16bccf12f6bb/tftpboot-installation-SLE-15-SP1-x86_64-14.427.1-3.6.2.noarch.rpm
/var/spacewalk/packages/NULL/d6a/tftpboot-installation-SLE-15-SP3-x86_64/16.56-1.2/noarch/d6ae119e267304e6c8234f146e15ce86fbd2aae11f0a37893f8be83710191fd8/tftpboot-installation-SLE-15-SP3-x86_64-16.56-1.2.noarch.rpm
/var/spacewalk/packages/NULL/af1/tftpboot-installation-SLE-15-SP2-x86_64/14.470-1.1/noarch/af112e61b04006f766c3558cdc568f352e32c8ecba52c9f71bb14551ed22083d/tftpboot-installation-SLE-15-SP2-x86_64-14.470-1.1.noarch.rpm
```